Formula/sniffglue.rb

```diff
@@ -0,0 +1,26 @@
+class Sniffglue < Formula
+  desc "Secure multithreaded packet sniffer"
+  homepage "https://github.com/kpcyrd/sniffglue"
+  url "https://github.com/kpcyrd/sniffglue/archive/v0.10.1.tar.gz"
+  sha256 "27b9448a317957eface67fab1a264057b03b614b588031e4f47655a8e369ead7"
+
+  depends_on "rust" => :build
+
+  resource "pcap" do
+    url "https://github.com/kpcyrd/sniffglue/raw/163ca299bab711fb0082de216d07d7089c176de6/pcaps/SkypeIRC.pcap"
+    sha256 "bac79a9c3413637f871193589d848697af895b7f2700d949022224d59aa6830f"
+  end
+
+  def install
+    system "cargo", "install", "--locked", "--root", prefix, "--path", "."
+
+    etc.install "sniffglue.conf"
+    man1.install "docs/sniffglue.1"
+  end
+
+  test do
+    resource("pcap").stage {
```

Comment:
`testpath.install resource("testdata")` is probably better. Check `abyss.rb` for usage of resources in the test.

Reply:
Thanks, done!